service/index.js

```diff
@@ -24,6 +24,12 @@ module.exports = {
     return pool.query(q);
   },
 
+  deactivateUser(email) {
+    let q = `select up_user_deactivate('${email}')`;
```

Comment:
I'm curious about the naming convention, why is it up_user?

Reply:
this is how function and stored procedures are often named in the industry. up stands for 'user procedure'. Where i work we use up_NAME_HERE for stored procedures and fn_NAME_HERE for functions, but thats sql server. postgres is a bit different, and functions in postgres are like stored procedures in sql server. I looked it up and the convention is still to use up_... i believe. hope this helps!